pkgs/applications/version-management/gitea/default.nix

```diff
@@ -19,12 +19,12 @@
 
 buildGoModule rec {
   pname = "gitea";
-  version = "1.19.2";
+  version = "1.19.3";
 
   # not fetching directly from the git repo, because that lacks several vendor files for the web UI
   src = fetchurl {
     url = "https://dl.gitea.io/gitea/${version}/gitea-src-${version}.tar.gz";
```

Comment:
Could you update this to `dl.gitea.com`, there is a redirect in place from .io to .com, but using .com directly would probably be best as we may eventually have to decomm the legacy CDN.

Reply:
```suggestion
    url = "https://dl.gitea.com/gitea/${version}/gitea-src-${version}.tar.gz";
```